pandas/_libs/tslibs/parsing.pyx

```diff
@@ -30,6 +30,8 @@ from dateutil.relativedelta import relativedelta
 from dateutil.parser import DEFAULTPARSER
 from dateutil.parser import parse as du_parse
 
+from pandas._config.config import get_option
```

Comment:
rather than reaching in, I would expose get_options, and company in pandas._config (in the 'all')?